ne/README.md

```diff
@@ -2,17 +2,17 @@
 
 Computers are common in today's world, as they are able to perform a wide variety of tasks quickly and accurately. They are used in many different industries, such as business, healthcare, education, and entertainment, and have become an essential part of daily life for many people. Besides this, they are also used to perform complex scientific and mathematical calculations, to store and process large amounts of data, and to communicate with people around the world.
 
-कम्प्युटरहरू आजको संसारमा सामान्य छन्, किनकि तिनीहरूले विभिन्न प्रकारका कार्यहरू छिटो र सही रूपमा गर्न सक्षम छन्। तिनीहरू विभिन्न उद्योगहरू, जस्तै व्यवसाय, स्वास्थ्य सेवा, शिक्षा र मनोरन्जन, र धेरै व्यक्तिहरूको लागि दैनिक जीवनको आवश्यक हिस्सा भएको छ। यस बाहेक, तिनीहरू जटिल वैज्ञानिक र गणितीय गणनाहरू प्रदर्शन गर्न, ठूला मात्रामा डाटा भण्डार गर्न र प्रक्रिया गर्न, र विश्वभरका मानिसहरूसँग कुराकानी गर्न सक्छन।
+कम्प्युटर, आज को समयमा सबैले हरेक दिन सुनिरहने र प्रयोग गरी रहने साधन, बिसेगरी त्यसको छिटो र सही रुपमा काम गर्न सक्ने क्षमताको करण। कति प्राय मनिसहरुको लागि त झन कम्प्युटर जीवन जिउने माध्यम नै बनिसकेको छ। यस्को बास्तविक परिचय त त्यति बेला हुन्छ जब हामी यसलाई हामीले सोच्न सक्ने सम्म हरेक ठाउँमा  पाउछाैँ। चाहे त्यो अस्पतालको गम्भीर कामको लागी होस या त घरमा मनोरञ्जनात्मक कार्य को लागी होस, यस्को जातिल कार्य र ठूला डाटा प्रक्रिया सेकेन्डमा नै गर्न सकने क्षमताले आज कम्प्यूटर हरेक गोजीहरुमा हटाएका छ। तर यो कम्प्युटर आफ्नैमा कसरी काम गर्छ? के तपाई यो सोच्दै हुनुहुन्छ? यो बुझ्नाको लागी कम्प्यूटरलाई हामी हाम्रो दिमागसँग तुलना गरी हर्ओँ हैता।
```

Comment:
I think we have couple typos here : 
`बिसेगरी` : विशेष`
`जातिल ` : `जटिल `
`हटाएका `: `अटाएका`
`हर्ओँ हैता।`: `हेरौ  है त `